flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/internal/JdbcBatchingOutputFormat.java

```diff
@@ -179,11 +178,9 @@ public synchronized void flush() throws IOException {
 					throw new IOException(e);
 				}
 				try {
-					if (!connection.isValid(connectionCheckTimeoutSeconds)) {
```

Comment:
Why remove this?

Reply:
in this class i dont find any object can get  connectionCheckTimeoutSeconds

Reply:
We shouldn't remove this condition, otherwise, it will reestabilish connection when every attempt is failed. 

Maybe we can move `connectionCheckTimeout` from `JdbcOptions` to `JdbcExecutionOptions`, then you can get the value here.